runbotics/runbotics-orchestrator-ui/src/src-landing/translations/pl/footer.json

```diff
@@ -0,0 +1,3 @@
+{
+    "Landing.Footer.ScrollTop": "do góry"
```

Comment:
not sure about this translation

Reply:
" przewiń do góry" is a bit long.

maybe "do początku" / "początek strony" / "góra strony"?